core/request.py

```diff
@@ -87,15 +92,20 @@ def Get(url, headers=headers):
         verbout(R, 'Error: Invalid URL Format')
         ErrorLogger(url, e.__str__())
         return None
+    except requests.exceptions.ReadTimeout as e:
+        verbout(R, 'Exception at: '+color.GREY+url)
+        verbout(R, 'Error: Read Timeout. Consider increasing the timeout value via --timeout.')
+        ErrorLogger(url, e.__str__())
+        return None
     except requests.exceptions.HTTPError as e:  # if error
-        verbout(R, "HTTP Error : "+main_url)
-        ErrorLogger(main_url, e.__str__())
+        verbout(R, "HTTP Error Encountered : "+main_url)
+        ErrorLogger(url, e.__str__())
```

Comment:
Ah this was a big mistake which I managed to overlook. :(

Reply:
Yeah, I fixed it. Always there for you, bruh.